docs/src/docs/modules/layout-check.adoc

```diff
@@ -49,26 +45,24 @@ In order to get the layout check to work, you need at least a reference image an
 ----
 The directory for reference image may result in `src/test/resources/layout-references/chrome/WelcomePage.png` for example.
 
-== PIXEL Mode
-
 The comparison is generally carried out over the entire reference image. In this case, it is a prerequisite that the reference screenshot and the screenshot created during the test runtime have the same resolution (height and width are identical).
 
-Mostly the website consists of many more elements than should be checked. Ingredients such as the header,
-menus or footer may falsify the test result if there are deviations from the reference,
-but these are not currently the subject of the test. For this reason,
-the `{layoutcheck_ignore_color}` property can be used to determine that a particular color in the image can be excluded from comparison.
+To control the threshold, you can use the following properties.
```

Comment:
How does these properties work together with the threshold parameter of `assertScreenhot()` of `LayoutCheck` and `GuiElement.asserts()`? 
The parameter are not optional. Under which conditions a test is passed or failed if both (property and parameter) are defined?

BTW: The parameter defines a value in percent. This is not described.